core/src/main/scala/com/karumi/shot/reports/ConsoleReporter.scala

```diff
@@ -0,0 +1,65 @@
+package com.karumi.shot.reports
+
+import com.karumi.shot.base64.Base64Encoder
+import com.karumi.shot.domain._
+import com.karumi.shot.ui.Console
+
+class ConsoleReporter(console: Console) {
```

Comment:
I'd merge these two public methods into a single method we could parametrize to indicate if showing the base 64 error is needed or not. This could simplify the usage and provide the responsibility of rendering the error or not to this class and not the user of the class.

Reply:
Yes, good point ⚫️